tests/test_query.py

```diff
@@ -15,11 +15,40 @@ def test_data():
             "dataset": "http://neurobagel.org/vocab/qpn",
             "dataset_name": "QPN",
             "num_matching_subjects": 50,
+            "subject_file_paths": [
+                "/my/happy/path/sub-0653/to/session-02",
+                "/my/happy/path/sub-1063/to/session-01",
```

Comment:
How do you pick the number of paths here? The `num_matching_subjects` says 50. Not suggesting you put 50 paths here, rather that you adjust `num_matching_subjects` down to a correct value. But the numbers should match up. I would go for something like 4 paths max to keep this readable. The minimum amount of data that passes the tests.